modules/create-aws-cluster.adoc

```diff
@@ -73,6 +71,15 @@ CIDR configurations cannot be changed later. Confirm your selections with your n
 If the cluster privacy is set to *Private*, you will not be able to access your cluster until you configure private connections in your cloud provider.
 ====
 
+. Optional: Select *Enable etcd encryption* if you require etcd key value encryption. This is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
++
+[IMPORTANT]
+====
+By enabling etcd encryption for the key values in etcd state, you might incur a performance overhead of approximately 20%. Red Hat only recommends that you enable etcd encryption in addition to the default storage-level encryption if you specifically require this for your use case.
+====
+
+. Select *Enable user workload monitoring* to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. The option is enabled by default.
```

Comment:
Interesting - in the OCM web UI, this is "Enable user workload monitoring", but in the ROSA CLI this is "Disabled user worload monitoring".

We went with the "Disabled" version in ROSA since user workload monitoring is enabled by default, and we want to give users the option of toggling it away from its default state.

How can we reconcile the documentation so that it can be reused across interfaces? Possible options:
- Change ROSA CLI to "Enable uwm"
- Change OCM web UI to "Disable uwm"
- Change docs to something generic like "The *user workload monitoring* option allows you to [...]. User workload monitoring is enabled by default"

Reply:
@vkareh thank you for your review and for highlighting this.

@0kashi what is your view on how to address this? Thanks!

Reply:
I agree with you @pneedle-rh.  Since the differences already exist between the CLI and UI (and changing those to match are more effort), I think your suggestion works. I like using "Leave" as opposed so "select" since it is already selected and won't confuse customers.

Reply:
@0kashi thanks for your input. I have subsequently updated the line to "Leave **Enable user workload monitoring** selected to monitor your own projects...".